v19.2/cockroachcloud-connect-to-your-cluster.md

```diff
@@ -94,7 +94,7 @@ On the machine from which you want to connect to your cluster:
 
     The **Connect** tab is displayed.
 
-    <img src="{{ 'images/v19.1/cockroachcloud/connect-tab.png' | relative_url }}" alt="Connect to cluster" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v19.2/cockroachcloud/connect-tab.png' | relative_url }}" alt="Connect to cluster" style="border:1px solid #eee;max-width:50%" />
 
 6. Select a connection option:
```

Comment:
The order of the tabs on the connection modal has recently been updated (CockroachDB Client is the default now).